### PR TITLE
output time duration per test case

### PR DESCRIPTION
**Pull request by edisonxiang**
**What type of PR is this?**
/kind test

**What this PR does / why we need it**:
output the time duration per test case for the E2E tests.
It can be used to measure the pod starup time and so on.

### CODE REVIEW

File: tests/e2e/deployment/deployment_test.go [hunk 1]
```diff
@@ -76,8 +78,14 @@ var _ = Describe("Application deployment test in E2E scenario", func() {
 				}
 			}
 			utils.CheckPodRunningState(ctx.Cfg.ApiServer+AppHandler, podlist)
+			// End test timer
+			testTimer.End()
+			// Print result
+			testTimer.PrintResult()
```
Comment (m1093782566) [inline tests/e2e/deployment/deployment_test.go]: Do we need to store the time duration in somewhere?

Comment (edisonxiang) [reply tests/e2e/deployment/deployment_test.go]: OK. Thanks

Comment (edisonxiang) [reply tests/e2e/deployment/deployment_test.go]: Done, PTAL. Thanks

**Comment by edisonxiang** [comment]
@sids-b It makes sense. Thanks. Updated:)

**Pull request merged**